src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package-lock.json

```diff
@@ -7153,10 +7157,13 @@
       "dev": true
     },
     "oidc-client": {
-      "version": "1.9.0-beta.3",
-      "resolved": "https://registry.npmjs.org/oidc-client/-/oidc-client-1.9.0-beta.3.tgz",
-      "integrity": "sha512-8JqmesN7n7sQrdeEYiuWyWjNp/6cZGOHU5IaYrULm/qdYo6cAheeZl93bDkob6MdfbQMsQ2HGG/+EOHFdFQcSQ==",
+      "version": "1.9.0-beta.4",
+      "resolved": "https://registry.npmjs.org/oidc-client/-/oidc-client-1.9.0-beta.4.tgz",
+      "integrity": "sha512-BE6o220w9kDTicmsX324Z5t3Gg20G9NgQjCOFP6SUY7glJYBohyw9DVPt/khPmmg+wabufJq1tH2zyEadMf7Pw==",
       "requires": {
+        "base64-js": "^1.3.0",
+        "core-js": "^2.6.4",
+        "crypto-js": "^3.1.9-1",
```

Comment:
Just to confirm: @Pilchie has approved the added transitive package dependencies (and increased visibility of the base64-js dependency) too?